src/components/shared/ButtonsLarge/index.tsx

```diff
@@ -0,0 +1,34 @@
+import classnames from 'classnames';
+
+export enum ButtonLargeVariants {
+  Primary = 'primary',
+  Outline = 'outline',
+}
+
+interface ButtonLargeProps {
+  variant: string;
+  text: string;
+  hasIcon: boolean;
+  disabled?: boolean;
+  onClick: () => unknown;
+}
+
+const ButtonLarge = ({ variant, text, hasIcon, onClick, disabled }: ButtonLargeProps) => {
```

Comment:
Following my comment regardint extending the html button type.. If you apply my first suggestion here you would have something like
```suggestion
const ButtonLarge = ({ variant, hasIcon, children, ...rest }: ButtonLargeProps) => {
```